### PR TITLE
Formatting, Linting and pep8 changes for galaxy_importer ans ansible-…

### PR DESCRIPTION
**Pull request by anuj-apdev**
…test sanity

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: meta/runtime.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-requires_ansible: ">=2.11"
```
Comment (utkarsh5474) [inline meta/runtime.yml:2]: Is this intended?

File: roles/kafka_broker/tasks/main.yml [hunk 2]
```diff
@@ -351,7 +351,17 @@
     owner: root
     group: root
   notify: restart kafka
-  diff: "{{ not mask_sensitive_diff|bool }}"
+  diff: {{ not mask_sensitive_diff|bool }}
+
+- name: Create sysctl directory on Debian distributions
```
Comment (utkarsh5474) [inline roles/kafka_broker/tasks/main.yml:356]: Is this related to ansible-sanity failure?

File: roles/kafka_broker/tasks/secrets_protection.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+- name: Create Kafka Broker Config with Secrets Protection
```
Comment (utkarsh5474) [inline roles/kafka_broker/tasks/secrets_protection.yml:2]: Same as above. Looks unrelated.

File: roles/kafka_rest/tasks/main.yml [hunk 4]
```diff
@@ -138,7 +138,13 @@
     owner: "{{kafka_rest_user}}"
     group: "{{kafka_rest_group}}"
   when: rbac_enabled|bool
+<<<<<<< Updated upstream
```
Comment (utkarsh5474) [inline roles/kafka_rest/tasks/main.yml:141]: Lots of git leftovers in this task

File: roles/kafka_broker/tasks/main.yml [hunk 5]
```diff
@@ -351,7 +351,17 @@
...
+  file:
+    path: /usr/lib/sysctl.d
+    state: directory
+  when: ansible_distribution == "Debian"
+  tags:
+    - sysctl
+    - privileged
+>>>>>>> Stashed changes
```
Comment (utkarsh5474) [inline roles/kafka_broker/tasks/main.yml:364]: git leftover

File: meta/runtime.yml (hunk 1 above)
Comment (anuj-apdev) [reply meta/runtime.yml:2]: Yes, This file is needed. However i am still testing this with 2.9 onwards.
So far i have made it work just upto 2.11+.

File: roles/kafka_broker/tasks/main.yml (hunk 2 above)
Comment (anuj-apdev) [reply roles/kafka_broker/tasks/main.yml]: It was a warning during galaxy_import command. I have rolled back these changes now.

File: roles/kafka_broker/tasks/secrets_protection.yml (hunk 3 above)
Comment (anuj-apdev) [reply roles/kafka_broker/tasks/secrets_protection.yml]: Rolled back. Was a part of merge conflict as i pulled my changes from 7.1.x to 7.0.0.

File: roles/kafka_broker/tasks/main.yml (hunk 2 above)
Comment (utkarsh5474) [reply roles/kafka_broker/tasks/main.yml]: I meant the new task `Create sysctl directory on Debian distributions`

**Comment by nsharma-git** [comment]
This has been merged as part of https://github.com/confluentinc/cp-ansible/pull/1071
